examples/howto/marginalizing-models.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>Add labels and title to plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/628/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
None of the other histograms for the gmm have labels and title. What labels and title do you suggest?